benchmarks/java/pom.xml

```diff
@@ -0,0 +1,148 @@
+
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2011 Google Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+<!--  <parent>
+	  <groupId>com.google.protobuf</groupId>
+	  <artifactId>protobuf-parent</artifactId>
+	  <version>3.5.0</version>
+  </parent> 
+
+  <packaging>pom</packaging> -->
+  <artifactId>protobuf-java-benchmark</artifactId>
+  <groupId>com.google.protocolbuffers</groupId>
+  <version>1.0.0</version>
+  <name>Protocol Buffers [Benchmark]</name>
+  <description>
+	  The benchmark tools for Protobuf Java.
+  </description>
+
+  <dependencies>
+    <dependency>
+	    <groupId>com.google.protobuf</groupId>
+	    <artifactId>protobuf-java</artifactId>
+	    <version>3.5.0</version>
+	</dependency>
+	<dependency>
+		<groupId>com.google.caliper</groupId>
```

Comment:
Please format this file. The indentation is incorrect.

Reply:
Done